tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "strict": true,
     "jsx": "preserve",
     "lib": ["dom", "ES2022"],
-    "target": "es5",
+    "target": "ES2022",
```

Comment:
This was necessary to make `tsc` not complain about using a top-level `await` in the new script.